src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -12,92 +12,187 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
+
+### Patches versions
+
+There should only have regressions issues in the kanban of a patch version. For example, 1.7.7.1 and 1.7.7.2 (and so on...) kanbans only includes the regressions of the 1.7.7.0 version.
+
+### Minor versions
+
+There should only have "must-have" issues in the kanban of a minor/major version and it always have an owner, be it a product manager or a developer.
+Only "must-have" issues are included in the kanban, as it must reflect the progress of the version. For example, an issue reports a must-have bug prioritized for the 1.7.8.0, the issue must appear in the 1.7.8 kanban while the linked pull request must have the 1.7.8.0 milestone.
+
+### Nice to have
+
+"Nice to have" issues are part of another [kanban](https://github.com/PrestaShop/PrestaShop/projects/13) and are not intended for a particular version.
+
+## Definition of states
+
+Community contributions that fix a feature request or an improvement, must be approved by the Core product team if they bring a change in the functional behavior and by the Core developer team if they bring a change in the technical behavior. Ideally before being reviewed, at least before being merged. Here is why:
+
+It is a bug? The specifications must be completed only when it's incomplete or when there's a behavior change.
+It is a feature request or an improvement? It must be validated, prioritized, and specified. Specification can be written by a contributor or by the Core Product team. The Core Product team has the final word in whether the specification is valid or not.
+
+A pull request that shows no activity for more than three months must be closed. If it is fixing a major bug or any topwatcher issue, it will be back in the to do column again to be completed by a developer.
+
+### Definition of ready
+
+An important notion to keep in mind is the definition of ready. Several elements must be prepared upstream to get the issue ready to be worked on, otherwise it will not be included in the backlog.
+
+An issue is ready when it contains:
+- an explicit title and description
+- validated & available specifications
+- validated & available mockups
+- validated & available wording
+- validated & available acceptance tests
+
+#### Not ready
+
+**All the must-have issues that should be included in this version** but that are not yet ready - need to be assigned to an owner, be it a Product Manager or a Lead Developer.
+
+_Who adds issues to this column?_ It is the core product team members once they choosed to prioritize the issue as a "must-have", but it could also be the lead developer with the approval of a Product Manager.
+
+#### Backlog
+
+**All issues that are considered _ready_** by the Product Managers and the lead dev. Issues in this column will be discussed with the core maintainers.
```

Comment:
```suggestion
**All issues that are considered _ready_** by the Product Managers and the Lead Developer. Issues in this column will be discussed with the core maintainers.
```
Lead Developers ? Is it just @eternoendless or do @PierreRambaud and me qualify ? 😄